src/context/fetch.ts

```diff
@@ -26,6 +28,23 @@ const createFetchRequestParameters = (input: MockedRequest): RequestInit => {
     return requestParameters
   }
 
+  // When the user made their original request, the browser added a boundary e.g.
+  // "content-type": multipart/form-data; boundary=----WebKitFormBoundarygyGKnRF8C9LT0BhB"
+  // If we now reuse this string as a user-provided header, the form-data will break and no payload will be send
+  // https://community.cloudflare.com/t/cannot-seem-to-send-multipart-form-data/163491/2
+  if (
+    input.headers.get('content-type')?.includes('multipart/form-data') &&
+    typeof body === 'object'
+  ) {
+    input.headers.delete('content-type')
+    const formData = new FormData()
```

Comment:
If we make the fix in `fetch.ts`, we need to create fresh form data, which was not necessary in `mockServiceWorker.js`. Previous commit: https://github.com/mswjs/msw/pull/1188/commits/e00de1e4b8bafb2dbbe29c0ffc500daf7b3c3246.
 
I would prefer doing the fix in` mockServiceWorker.js` as seen in previous commit, but I understand the concern about breaking changes